### ISSUE TITLE
Cannot upload sourcemaps since updating the plugin: plugin incorrectly identifies Sentry CLI version

### ISSUE THREAD

**Issue by dkarter** [getsentry/sentry-fastlane-plugin#76]
### Environment

**How do you use Sentry?**
Sentry SaaS (sentry.io)

**Which SDK and version?**
React Native ("@sentry/react-native") 2.2.1
sentry-fastlane-plugin 1.8.1
Sentry CLI 1.63.1

### Steps to Reproduce

1. Ran the sentry_upload_file action after updating sentry-fastlane-plugin

### Expected Result

Source & Sourcemap are uploaded.

### Actual Result
Error about about Sentry CLI being out of date, even though I upgraded it and the version matches what the error says it should be:

```
[!] Your sentry-cli is outdated, please upgrade to at least version 1.63.1 and start your lane again!
```

![image](https://user-images.githubusercontent.com/551858/109907997-1921fc00-7c69-11eb-816e-99ebe1758010.png)

How it's configured:
<img width="1067" alt="image" src="https://user-images.githubusercontent.com/551858/109887386-36de6980-7c47-11eb-8180-c6c0c035d2c6.png">

**Comment by philipphofmann** [comment]
Hey @dkarter, 

that's weird. It works for me when I try it locally. Can you maybe try to reload your shell with something like `exec "$SHELL"` and try again?

I don't know your local setup, but it could be that fastlane is looking up the sentry-cli from a different place. Can you maybe try something like this and make sure it's pointing to the correct sentry-cli version.
```
lane :sentry-info do
  sh("sentry-cli --version")
  sh("which sentry-cli")
end
```

**Comment by dkarter** [comment]
Thank you @philipphofmann, really appreciate your help! I was able to resolve the issue thanks to your tip!

I had two versions of sentry-cli installed, one through the root of the project (where the shared action was running from), and another globally installed via homebrew.

When I ran the lane from the `ios/` folder I was getting the up-to-date homebrew version, but when I added those calls to the shared custom action, which is executed via `yarn` from the root of the project, it was picking up the the `node_modules` version of sentry-cli which was out of date.
